var/spack/repos/builtin/packages/sirius/package.py

```diff
@@ -48,6 +50,7 @@ class Sirius(CMakePackage, CudaPackage):
 
     variant('shared', default=True, description="Build shared libraries")
     variant('openmp', default=True, description="Build with OpenMP support")
+    variant('boost_filesystem', default=False, description="Use boost filesystem")
```

Comment:
Could you maybe explain "why" someone might want it?

Reply:
Yes!

Reply:
thanks, awesome!